FRW/Services/Manager/TransactionManager.swift

```diff
@@ -224,6 +224,14 @@ extension TransactionManager {
                             self.errorMsg = result.errorMessage
                             self.internalStatus = .failed
                             debugPrint("TransactionHolder -> onCheck result failed: \(result.errorMessage)")
+                            
+                            switch result.errorCode {
+                            case .storageCapacityExceeded:
+                                let data = InsufficientStorageTransactionFailureData(minimumBalance: WalletManager.shared.minimumStorageBalance)
+                                NotificationCenter.default.post(name: .insufficientStorageTransactionFailure, object: data)
```

Comment:
Cause we post the `transactionStatusDidChanged` event already, we should reuse the same notification

Reply:
THis is removed. The alert view is inserted in the top view controller, wrapped in a UIHostincController